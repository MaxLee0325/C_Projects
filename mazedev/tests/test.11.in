Alpha
Bravo
END
END
Alpha
Bravo
Charlie
END
