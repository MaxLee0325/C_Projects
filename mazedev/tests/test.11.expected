Entering room Alpha.
Oops: Room Alpha does not connect to room Bravo.
